jms/1.8/modules/ROOT/pages/jms-troubleshooting.adoc

```diff
@@ -78,6 +78,17 @@ To resolve this scenario:
 
 Implement a fallback solution by managing the life cycle of containers to detect when the primary node fails and starts a new node.
 
+== Troubleshoot Prefetch
+
+Using prefetch can result in the loss of predictability of API calls. If prefetch is used, JMS Connector aggressively tries to fetch messages, which results in increased API calls. If predictability and control on API requests is important, check your third-party JMS vendor provider documentation to disable prefetch.
```

Comment:
After discussing with @nicomz we decided to use a different approach to avoid confusions:
1- create a known issue related to prefetch+ack manual+concurrent consumes
2- consume documentation should have a link to troubleshooting prefetch
3- troubleshooting prefetch should have alternatives for the broken use case